bitnami/geode/templates/locator/statefulset.yaml

```diff
@@ -0,0 +1,387 @@
+      {{- if .Values.locator.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.locator.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.locator.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.locator.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.locator.priorityClassName }}
+      priorityClassName: {{ .Values.locator.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.locator.schedulerName }}
+      schedulerName: {{ .Values.locator.schedulerName }}
+      {{- end }}
+      {{- if .Values.locator.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.locator.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if or .Values.volumePermissions.enabled .Values.locator.initContainers }}
+      initContainers:
+        {{- if .Values.volumePermissions.enabled }}
+        - name: volume-permissions
+          image: {{ include "geode.volumePermissions.image" . }}
+          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          command:
+            - /bin/bash
+          args:
+            - -ec
+            - |
+              find /bitnami/geode -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.containerSecurityContext.runAsUser}}:{{ .Values.podSecurityContext.fsGroup }}
+          {{- if .Values.locator.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.locator.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.volumePermissions.resources }}
+          resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: data
+              mountPath: /bitnami/geode
+        {{- end }}
+        {{- if .Values.locator.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.locator.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
+      containers:
+        - name: geode
+          image: {{ include "geode.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.locator.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.locator.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.locator.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.locator.command "context" $) | nindent 12 }}
+          {{- else if .Values.auth.tls.enabled }}
+          command:
+            - /bin/bash
```

Comment:
A minor thing, no changes requested: Instead of having the newline, we could just do:

```yaml
command: ["/bin/bash"]
```

Reply:
Thanks, I'd rather keep it as it's.

Reply:
Sure, it's fine